rules/configuration.smk

```diff
@@ -337,6 +337,10 @@ else:
         sample=any_given('sample', 'condition')
 
 
+def get_workflow():
+    return workflow.snakefile.split('/')[-2]
+
+
```

Comment:
This is fine for now. I implemented sth like this as well in trackhub. I think however this will bite us in the ass later. Probably we should put it in some schemas or sth.